src/Language/Go/Assignment.hs

```diff
@@ -290,7 +290,7 @@ defaultCase :: Assignment (Term Loc)
 defaultCase = makeTerm <$> symbol DefaultCase <*> children (Go.Syntax.DefaultPattern <$> (expressions <|> emptyTerm))
 
 defaultExpressionCase :: Assignment (Term Loc)
-defaultExpressionCase = makeTerm <$> symbol DefaultCase <*> (Go.Syntax.DefaultPattern <$ rawSource <*> (expressions <|> emptyTerm))
+defaultExpressionCase = makeTerm <$> symbol DefaultCase <*> children (Go.Syntax.DefaultPattern <$> (expressions <|> emptyTerm))
```

Comment:
Lots of misc assignment fixes just so that we can appropriate match a la carte and precise tag output.